upstash/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fe1f17e3-11a4-4e44-b819-8781ebcc86f8",
+  "app_id": "upstash",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Publish your Upstash Redis metrics to Datadog",
+    "title": "Upstash",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
Can you please update this screenshot so it shows the latest changes to your dashboard?

Reply:
screenshoot updated